src/movie-components/movie-card-component/_movie-card.scss

```diff
@@ -0,0 +1,38 @@
+.movie-card {
+  height: 500px;
+  background: lightblue;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  overflow: hidden;
+  box-shadow: 0 0 2px 2px black;
+  border-radius: 5px;
+
+  h1 {
```

Comment:
Awesome work implementing this Sass nesting! It really brings some order to our files.